Virtual Librarian/VirtualLibrarian.API/Controllers/BooksController.cs

```diff
@@ -12,18 +12,17 @@ namespace VirtualLibrarian.API.Controllers
 {
     public class BooksController : ApiController
     {
-        private static Library library = null;
-        private static readonly object padLock = new object();
-        public BooksController()
+        private readonly ILibrary _library; 
+        public BooksController(ILibrary library)
         {
-            library = Library.Instance;
+            _library = library;
         }
 
         [HttpGet]
         [Route("GetAvailableBooksSorted")]
         public IHttpActionResult GetAvailableBooksSorted()
         {
-            List<Book> sortedAvailableBooks = library.GetAvailableBooksSorted();
+            List<Book> sortedAvailableBooks = _library.GetAvailableBooksSorted();
```

Comment:
use `var`

Reply:
Also, I thing you should use some sort of limit for books, for pagination or something similar